plugin.video.youtube/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
+<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
```

Comment:
Yeah, the plan is definately to migrate the stuff except for testing off kolifanes in the next version.

> On May 17, 2016, at 9:27 AM, Razzeee <notifications@github.com> wrote:
> 
> In plugin.video.youtube/addon.xml <https://github.com/xbmc/repo-plugins/pull/480#discussion_r63520850>:
> 
> > @@ -1,5 +1,5 @@
> >  <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
> > -<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
> > +<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
> Well, it's up to you what you do. But I would choose a construct, that is simple and thus easily understandable. You don't want to confuse potential committers or people interested in the addon. Or render them unable to find your repo ;)
> If you go for it, it's probably nothing that has to be done in this release.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r63520850>